.github/workflows/e2e.yml

```diff
@@ -50,6 +50,7 @@ on:
         - vcluster leg 6
         - vcluster leg 7
         - leg 8
```

Comment:
I see leg 8 doesn't have a "vcluster" prefix, should we add that?

Reply:
I saw that too. But leg 8 is special in that it does both admintools and vcluster. It will start as admintools, do an upgrade and end up as vcluster. So, it's kind of both.

Reply:
Agreed.